samples/net/echo_client/overlay-qemu_cortex_m3_eth.conf

```diff
@@ -0,0 +1,11 @@
+# Select Stellaris Ethernet
```

Comment:
Is there reason why we should not call this overlay-stellaris.conf?

Reply:
The other overlay files seem to following the naming convention overlay-board-device.conf, when the overlay file is specific to board.

Reply:
Ok, this makes sense.